docs/getting-started-guides/README.md

```diff
@@ -147,6 +153,7 @@ libvirt              | custom       | Fedora | flannel     | [docs](fedora/flann
 KVM                  | custom       | Fedora | flannel     | [docs](fedora/flannel_multi_node_cluster.md)      |          | Community ([@aveshagarwal](https://github.com/aveshagarwal))
 Mesos/Docker         | custom       | Ubuntu | Docker      | [docs](mesos-docker.md)                           | [✓][4]  | Community ([Kubernetes-Mesos Authors](https://github.com/mesosphere/kubernetes-mesos/blob/master/AUTHORS.md))
 Mesos/GCE            |              |        |             | [docs](mesos.md)                                  |          | Community ([Kubernetes-Mesos Authors](https://github.com/mesosphere/kubernetes-mesos/blob/master/AUTHORS.md))
```

Comment:
I realize it's not part of this PR, but I assume the OS should be Debian and the Networking should be custom or Docker? What sould config mgmt. be?